node_modules/col-core/lib/db.js

```diff
@@ -78,19 +78,323 @@ var init = module.exports.init = function(callback) {
 var setUpModel = function(sequelize) {
 
   /**
-   * The Course model
+   * The `canvas` table keeps track of the Canvas instances that Collabosphere should
+   * communicate with. It holds all the information that's needed to interact with the
+   * Canvas API and to embed on of the Collabosphere tools into Canvas
    *
-   * @property  {Number}      id                      The id of the course
+   * @property  {String}    app_domain      The domain where canvas is running on
```

Comment:
Should this be `canvas_api_domain`. I think Canvas is giving us `api_domain` instead of `app_domain`, and it would make it sound less like the "domain on which the Collabosphere app is running"